src/Pagination/PaginationManipulator.php

```diff
@@ -137,7 +153,7 @@ protected function registerPaginator(
         $paginatorFieldClassName = addslashes(PaginatorField::class);
 
         $paginatorType = PartialParser::objectTypeDefinition("
-            type $paginatorTypeName $modelClassDirective {
+            type $paginatorTypeName {$this->modelClassDirective()} {
```

Comment:
Just a tip:

If you use HEREDOC, you won't need to escape all that quotes.

Reply:
Let's discuss this in a seperate PR, that could be useful in a lot of places!